dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cs

```diff
@@ -1,3 +1,12 @@
+// ------------------------------------------------------------------------------
+// <auto-generated>
```

Comment:
This is NOT a generated file. It's artisanal hand-crafted code

Reply:
It's generated in the sense that it's copied into someone else's project, and they have no ability to modify it.